$OpenBSD: patch-panels_display_meson_build,v 1.3 2022/02/13 12:05:39 ajacoutot Exp $

Fix build with meson >= 0.60.0

No colord (needs GUdev) on OpenBSD (XXX add HAVE_COLORD to meson_options.txt)

Index: panels/display/meson.build
--- panels/display/meson.build.orig
+++ panels/display/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -44,7 +43,6 @@ sources += gnome.compile_resources(
 )
 
 deps = common_deps + [
-  colord_dep,
   gnome_desktop_dep,
   m_dep,
   upower_glib_dep
